hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -197,7 +197,7 @@ && isWithInRetryLimit(retryCount)) {
         if (!acceptAnyPrecheck) {
             final var expectedIngestStatus = getExpectedPrecheck();
             if (expectedIngestStatus != OK
-                    && !spec.setup().streamlinedIngestChecks().contains(expectedIngestStatus)) {
+                    && spec.setup().streamlinedIngestChecks().contains(expectedIngestStatus)) {
```

Comment:
Why should this switch? 
Also, won't this make other tests fail?

Reply:
No, this actually fixes the bug. 
Intended logic for this property was, if the mono-service has PreCheck expectation `X` other than OK, but new code has this as Handle expectation, this code just makes the code has hasPreCheckFrom(OK, X) by checking if the X is in `spec.streamlinedIngestChecks`.  Confirmed with @tinker-michaelj .

Reply:
Interesting, you fixed it here. I was following @rbair23 fix to use 
.hasPrecheckFrom(OK, x).hasKnownStatus(x) in the test suite. After your merge, I will remove my changes.